DataFormats/METReco/interface/CSCHaloData.h

```diff
@@ -69,6 +78,20 @@ namespace reco {
     edm::RefVector<reco::TrackCollection>& GetTracks(){return TheTrackRefs;}
     const edm::RefVector<reco::TrackCollection>& GetTracks()const {return TheTrackRefs;}
     
+    edm::RefVector<EcalRecHitCollection>& GetEBRechits(){return ecalebrhRefs;}
+    const edm::RefVector<EcalRecHitCollection>& GetEBRechits()const {return ecalebrhRefs;}
+    edm::RefVector<EcalRecHitCollection>& GetEERechits(){return ecaleerhRefs;}
+    const edm::RefVector<EcalRecHitCollection>& GetEERechits()const {return ecaleerhRefs;}
+
+    edm::RefVector<HBHERecHitCollection>& GetHBHERechits(){return hbherhRefs;}
+    const edm::RefVector<HBHERecHitCollection>& GetHBHERechits()const {return hbherhRefs;}
+
+    /*    edm::RefVector<HBHERecHitCollection>& GetHBRechits(){return hbrhRefs;}
+    const edm::RefVector<HBHERecHitCollection>& GetHBRechits()const {return hbrhRefs;}
+
+    edm::RefVector<HBHERecHitCollection>& GetHERechits(){return herhRefs;}
+    const edm::RefVector<HBHERecHitCollection>& GetHERechits()const {return herhRefs;}
+    */
```

Comment:
Is this commented out piece needed?
Please remove or add a comment why it is here.